plugins/modules/katello_manifest.py

```diff
@@ -106,7 +106,7 @@ def main():
                 if 'repository_url' in entity_dict:
                     params['repository_url'] = entity_dict['repository_url']
                 params.update(scope)
-                changed, result = module.resource_action('subscriptions', 'upload', params, options={'skip_validation': True}, files=files)
+                changed, result = module.resource_action('subscriptions', 'upload', params, files=files)
```

Comment:
We owe this (off topic) change to an improved apypie version, right?

Reply:
yes.